src/chart/funnel/FunnelView.ts

```diff
@@ -167,8 +169,141 @@ class FunnelPiece extends graphic.Polygon {
             stroke: visualColor
         });
     }
+
+    ratePiece: RatePiece;
 }
 
+class RatePiece extends graphic.Polygon {
+
+    constructor(data: SeriesData, idx: number) {
+        super();
+
+        const polygon = this;
+        const labelLine = new graphic.Polyline();
+        const text = new graphic.Text();
+        polygon.setTextContent(text);
+        this.setTextGuideLine(labelLine);
+
+        this.updateData(data, idx, true);
+    }
+    updateData(data: SeriesData, idx: number, firstCreate?: boolean) {
+        const polygon = this;
+        const layout = data.getItemLayout(idx);
+        const seriesModel = data.hostModel;
+        let opacity: number;
+        if (layout.isLastPiece) {
+            opacity = 1;
+        }
+        else {
+            opacity = 0.5;
+        }
+
+        if (!firstCreate) {
+            saveOldStyle(polygon);
+        }
+
+        if (layout.isLastPiece) {
+            polygon.useStyle(Object.assign(data.getItemVisual(idx, 'style'), { fill: 'rgba(0,0,0,0)' }));
```

Comment:
Can you explain what `fill: 'rgba(0,0,0,0)'` is used for?

Reply:
I want make overAll rate piece a transparent one.

Reply:
You can probably use `'none'`.

Reply:
ok